### DISCUSSION TITLE
Paramount+ won't allow login if Cookie Autodelete (and also uBlock Origin and NoScript) are enabled

### DISCUSSION THREAD

**Discussion by frustrated-user-1** [Cookie-AutoDelete/Cookie-AutoDelete#1635] [Q&A / Support]
This problem just started this week.
Can't log in to Paramount+ with extensions listed above enabled. Still can't log in if only Cookie Autodelete is enabled. Must disable all the above in order for website to work.
Ubuntu 24.04, Firefox 129.0. Cookie Autodelete 3.8.2

**Comment by pstemari** [comment]
Logging into a website generally requires cookies to be enabled. If you gray-list or white-list Paramount+, you should be able to login.

**Comment by kennethtran93** [comment]
Yes you will need to whitelist Paramount+, as well as allowing it through NoScript and uBlock Origin as well.
